_posts/2018-05-20-Y-The-Most-Beautiful-Idea-in-Computer-Science.md

```diff
@@ -0,0 +1,611 @@
+---
```

Comment:
Since the examples keep breaking in Sinon's docs, I've opened a discussion to make them runnable and have tests.

https://github.com/sinonjs/sinon/pull/1794

Feel free to use that for inspiration for your Jekyll project :)

Reply:
That's an amazing idea!

I've even added a task to my personal Trello board to try implementing an NPM module that does this 😄